kafka-agent/src/main/java/io/strimzi/kafka/agent/KafkaAgent.java

```diff
@@ -159,11 +162,17 @@ public void run() {
             boolean handleBrokerState() {
                 LOGGER.trace("Polling {}", brokerStateName);
                 boolean ready = false;
-                Integer running = Integer.valueOf(3);
-                Object value = brokerState.value();
+                Object observedState = brokerState.value();
 
-                if ((value instanceof Integer && running.equals(value))
-                        || (value instanceof Byte && running.equals(((Byte) value).intValue()))) {
+                boolean stateIsRunning = false;
+                if (observedState instanceof Integer) {
```

Comment:
Maybe I'm missing something, but the broker state is a byte value, so why not directly use `BrokerState` enum from Kafka, getting rid of `instanceof` checks and int conversions?

Reply:
Less dependencies?

Reply:
OK, you can use byte constants if you prefer, but the `observedState` will never be an int (unless Kafka changes the public enum), so this code is effectively unreachable.

Reply:
I guess Kate (or someone else if you volunteer 😉) would need to test it with the supported versions if any of them might use the integer part. It might be a leftover from Kafka 2.5.0. (This is also the reason why pulling in `BrokerState` is not a good idea. This stuff is not exactly public API which is not changing.)

Reply:
Sure, I'll do that.